Update 6.kill-query.md (#1143)
vesoft-inc/nebula-docs: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/6.kill-query.md

`KILL QUERY` can terminate the query being executed, and is often used to terminate slow queries.

!!! note

>     Users with the God role can kill all queries and users with other roles only can kill their own queries.

Reviewer: ```suggestion
    Users with the God role can kill any query. Other roles can only kill their own queries.
```
